xml/System.Runtime.InteropServices.Marshalling/Utf8StringMarshaller.xml

```diff
@@ -93,8 +103,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Frees native resources.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+<xref:System.Runtime.InteropServices.Marshalling.CustomTypeMarshallerFeatures.UnmanagedResources>
```

Comment:
Phantom xrefs in this file.

Reply:
Fixed https://github.com/dotnet/dotnet-api-docs/pull/8185/files/ab34e14794272db216d67e0d44c47155e27c1f31#r914257233